README.md

```diff
@@ -7,6 +7,49 @@ height="42" />
 
 Next-generation WordPress eCommerce Experience for WordPress sites at Newfold Digital.
 
+
+## Module Responsibilities
+
+* Ecommerce module displays `Home or Dashboard` page for Brand plugin, with below options:
+  1. Live site preview for both, live and coming soon mode for WordPress site with view & edit site options
+  2. Next steps to setup your site/store and also link to view completed tasks
+  3. Links to your Hosting Brand Account
+  4. Link to 24/7 support to reachout to experts to build your site for you. 
```

Comment:
This shouldn't be a concern of the e-commerce module, and we should strive to move them elsewhere.